blueprint/README.md

```diff
@@ -0,0 +1,49 @@
+# cloudify-interactive-tutorial
+
+## Summary
+
+Deploys Cloudify Interactive tutorial Application using cfy local on AWS.
+
+A bit more detailed explanation:
```

Comment:
wow... too detailed :)

Please remove up until "This blueprint consists of"...